R/inner_split.R

```diff
@@ -18,6 +18,11 @@
 #' `split_args$balance = NULL`.
 #' * For `boot_split` and `group_boot_split` objects, it will ignore
 #' `split_args$times`.
+#' * For `val_split`, `group_val_split`, and `time_val_split` objects, it will 
+#' interpret a length-2 `split_args$prop` as a ratio between the training and
+#' validation sets and split into inner analsysi and inner assessment set in 
```

Comment:
```suggestion
#' validation sets and split into inner analysis and inner assessment set in 
```

re: "inner assessment", this language is a bit :thinkies:. Is this a potato set or is this something else since we're supposedly doing validation?

Reply:
yes, the inner assessment set is the potato set. I wasn't sure where we were with the naming and I'm missing a way to differentiate what I call outer and inner analysis set and was therefore fully leaning into inner/outer naming here also also applying it to the assessment/potato set. 
You're not the only one though: https://github.com/tidymodels/rsample/issues/490